sdk/tools/cabman/CCFDATAStorage.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * COPYRIGHT:   See COPYING in the top level directory
+ * PROJECT:     ReactOS cabinet manager
+ * FILE:        tools/cabman/cabman.h
+ * PURPOSE:     Cabinet manager header
+ */
+
+#pragma once
+
+#include "cabinet.h"
+
+#ifndef CAB_READ_ONLY
+
+class CCFDATAStorage
+{
+public:
+    /* Default constructor */
+    CCFDATAStorage();
+    /* Default destructor */
+    virtual ~CCFDATAStorage();
+    ULONG Create();
+    ULONG Destroy();
+    ULONG Truncate();
+    ULONG Position();
+    ULONG Seek(LONG Position);
+    ULONG ReadBlock(PCFDATA Data, void* Buffer, PULONG BytesRead);
+    ULONG WriteBlock(PCFDATA Data, void* Buffer, PULONG BytesWritten);
+private:
+    char FullName[PATH_MAX];
```

Comment:
Fixed size buffer? Could this be replaced by a std::string?

Reply:
sure, but I don't want to refactor the entire thing, just the parts that I'll need to touch for my upcoming PR.
When that is done, feel free to go to town on the rest of cabman.